src/MoBi.Presentation/DTO/IndividualSelectionDTO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using MoBi.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.Presentation.DTO
+{
+   public class IndividualSelectionDTO
+   {
+      public IndividualSelectionDTO(IReadOnlyList<IndividualBuildingBlock> allIndividuals)
+      {
+         _allIndividuals.AddRange(allIndividuals);
+      }
+      
+      public static IndividualBuildingBlock NullIndividualBuildingBlock = new IndividualBuildingBlock().WithName(AppConstants.Captions.IndividualNotSelected);
```

Comment:
Similarily, I don't think this object belongs in a DTO. I would suggest to create a class and put all those statis members there). This class can also be static that way, there is no ambiguity